pages/15.contributing/04.tester/docs.md

```diff
@@ -70,6 +70,10 @@ mysql_version: "8.0"
 7. Remember to make sure you are using dev mode (index_dev.php on the end of the URL)
 8. If you make a mistake, open your Gitpod dashboard and delete the instance and start again.
 
+#### Resetting your local testing environment
+
+To quickly reset your local testing environment by deleting the DDEV containers without a database snapshot, removing the cache directory, and removing the `local.php` file you can run `ddev delete --omit-snapshot --yes && rm -rf var/cache && rm app/config/local.php`.
```

Comment:
```suggestion
To quickly reset your local testing environment by deleting the DDEV containers without a database snapshot, removing the cache directory, and removing the `local.php` file you can run `ddev delete --omit-snapshot --yes && rm -rf var/cache && rm app/config/local.php`. Note that from Mautic 5, the location of the local.php file is now `config/local.php`.
```

Reply:
I think we should add this comment as otherwise folks will be confused! Do you think it's best like this, or we leave a note about pre-Mautic-5 and have the Mautic 5 location as the default? Open to both, see what you think!